codegen/smithy-kotlin-codegen-testutils/src/main/kotlin/software/amazon/smithy/kotlin/codegen/test/ModelTestUtils.kt

```diff
@@ -173,7 +173,7 @@ fun Model.defaultSettings(
     sdkId: String = TestModelDefault.SDK_ID,
     generateDefaultBuildFiles: Boolean = false,
     nullabilityCheckMode: CheckMode = CheckMode.CLIENT_CAREFUL,
-    defaultValueSerializationMode: DefaultValueSerializationMode = DefaultValueSerializationMode.WHEN_DIFFERENT,
+    defaultValueSerializationMode: DefaultValueSerializationMode = DefaultValueSerializationMode.DEFAULT,
```

Comment:
This was originally set to `WHEN_DIFFERENT` because serializing defaults [caused problems with S3](https://github.com/smithy-lang/smithy-kotlin/pull/968), can we double-check those model issues are now resolved?